### PR TITLE
Added file system + inodes utilization metrics & updated tests

### PR DESCRIPTION
**Pull request by james-bebbington**
**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector/issues/847

**Description:** Added filesystem utilization metric & updated tests

### CODE REVIEW

File: receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_metadata.go [hunk 1]
```diff
@@ -45,12 +45,32 @@ var fileSystemUsageDescriptor = func() pdata.MetricDescriptor {
 	return descriptor
 }()
 
+var fileSystemUtilizationDescriptor = func() pdata.MetricDescriptor {
+	descriptor := pdata.NewMetricDescriptor()
+	descriptor.InitEmpty()
+	descriptor.SetName("system.filesystem.utilization")
+	descriptor.SetDescription("Filesystem utilization.")
+	descriptor.SetUnit("%")
```
Comment (bogdandrutu) [inline receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_metadata.go:53]: As discussed offline if you use % you need to multiply by 100 the result

**Comment by james-bebbington** [comment]
You guys approved this one, but presumably you have the same issue with `state` labels on this PR as with the cpu & memory utilization PRs?

**Comment by bogdandrutu** [comment]
Rebase?

**Comment by bogdandrutu** [comment]
@james-bebbington it looks like for this we do send percent separate for free/used/reserved :)))

**Comment by james-bebbington** [comment]
See https://github.com/open-telemetry/opentelemetry-collector/pull/1265#issuecomment-656490349

**Pull request closed without merging**